src/Geta.NotFoundHandler/Data/SqlSuggestionRepository.cs

```diff
@@ -120,21 +124,20 @@ public void Save(string oldUrl, string referer, DateTime requestedOn)
             _dataExecutor.ExecuteNonQuery(sqlCommand, requestedParam, refererParam, oldUrlParam);
         }
 
-        private DataTable GetAllSuggestionCount()
+        private DataTable GetAllSuggestions()
         {
-            var sqlCommand =
-                $"SELECT [OldUrl], COUNT(*) as Requests FROM {SuggestionsTable} GROUP BY [OldUrl] order by Requests desc";
+            var sqlCommand = $@"
+            SELECT c.[count] as OldUrlCount, c.OldUrl, r.Referer, count(r.Referer) as RefererCount
+            FROM
+                (SELECT 
+                    COUNT([OldUrl]) as [count]
+                    ,[OldUrl]
+                  FROM {SuggestionsTable}
+                GROUP BY [OldUrl]) c
+            INNER JOIN {SuggestionsTable} r  ON c.OldUrl = r.OldUrl
+            GROUP by c.[count], c.OldUrl, r.Referer
+            ORDER BY OldUrlCount desc";
```

Comment:
Some formatting of the SQL query is needed. Like `c.[count]` to `c.[Count]` and `count` function should be all capital - so that it consistent with other naming.

Reply:
Fixed